ui/app/models/pki-certificate.js

```diff
@@ -53,7 +53,7 @@ export default DS.Model.extend({
   ttl: attr({
     label: 'TTL',
     editType: 'ttl',
-    defaultValue: '720h',
+    defaultValue: '30h',
```

Comment:
another option for solving this would be to that would keep the `defaultValue`s set in the model and alleviate the need for computed properties in the component or conditionals in the template would be to overwrite `ttl` value that is extended inside the `pki-ca-certificate` model. i haven't tested it out but it's worth a shot! i think it's good to define values in the same source as much as possible. what are your thoughts?